evalml/data_checks/id_columns_data_check.py

```diff
@@ -187,51 +202,49 @@ def validate(self, X, y=None):
             key: value for key, value in id_cols.items() if value >= self.id_threshold
         }
 
-        first_col_id = False
-
-        if col_names and col_names[0] in id_cols_above_threshold:
-            first_col_id = True
-            del id_cols_above_threshold[col_names[0]]
-
         if id_cols_above_threshold:
-            warning_msg = ""
-            message_code = None
-            action_code = None
-            if first_col_id:
-                warning_msg = "The first column '{}' is most likely to be the ID column. Columns {} are also {}% or more likely to be an ID column"
+            if (
+                col_names[0] in id_cols_above_threshold
+                and id_cols_above_threshold[col_names[0]] == 1.0
+            ):
+                del id_cols_above_threshold[col_names[0]]
+                warning_msg = "The first column '{}' has a high likelihood of being the primary key"
```

Comment:
Nit: can we rephrase this to something like "The first column '{}' is likely to be the primary key"?